### PR TITLE
Define polygon outline border thickness

### PR DESCRIPTION
**Pull request by Rayffer**
**Purpose:**
This pull request aims to provide the possibility to specify a Radar Chart's polygon outline
#1277 

To specify outline, type the following code
```cs
    double[,] values = {
        { 5, 3, 10, 15, 3, 2 },
        { 5, 2, 10, 10, 1, 4 },
    };

    var radar = plt.AddRadar(values);
    radar.OutlineThickness = 3;
```

### CODE REVIEW

**Comment by bclehmann** [comment]
Hey, it looks like this PR is based off of a pretty out of date head (from mid-september). I think you need to rebase.

**Comment by Rayffer** [comment]
Done, thought I was up to date in my fork, thanks for the heads up!

File: src/ScottPlot/Plottable/RadarPlot.cs [hunk 1]
```diff
@@ -106,6 +106,8 @@ public class RadarPlot : IPlottable
         public int XAxisIndex { get; set; } = 0;
         public int YAxisIndex { get; set; } = 0;
 
+        public float OutlineThickness { get; set; } = 1;
```
Comment (bclehmann) [inline src/ScottPlot/Plottable/RadarPlot.cs:109]: This is a confusing name, especially with `LineWidth`, but I can't think of a better one. Maybe `DataThickness`?

Maybe Scott will have an idea.

**Review by bclehmann — APPROVED** [review]
Looks good, I don't know why the CI failed, but I don't think it had anything to do with you

File: src/ScottPlot/Plottable/RadarPlot.cs (hunk 1 above)
Comment (swharden) [reply src/ScottPlot/Plottable/RadarPlot.cs:109]: I'll split the difference and go with `OutlineWidth` 👍

**Comment by swharden** [comment]
Hi @Rayffer, thanks for this PR! Sorry for the long delay on this one - I'm in the middle of moving and the process has been a lot more disruptive than I anticipated it would be.

> Looks good, I don't know why the CI failed, but I don't think it had anything to do with you

Indeed! Re-running the job it appears all the tests passed this time

Thanks for your help along the way @bclehmann!

**Pull request merged**